### PR TITLE
Update reprex-dos-and-donts.Rmd

### PR DESCRIPTION
**Pull request by mrchypark**
vignettes typo

### CODE REVIEW

**Comment by batpigandme** [comment]
iff is actually notation for "if and only if" https://en.wikipedia.org/wiki/If_and_only_if

**Comment by mrchypark** [comment]
@batpigandme omg.. my bad. anyway, It's good ideas to make that explicit. I'll update it.

File: vignettes/reprex-dos-and-donts.Rmd [hunk 1]
```diff
@@ -80,7 +80,7 @@ The reprex code:
 
   * Should be easy for others to digest, so **they don't necessarily have to run it**. You are encouraged to include selected bits of output. :scream:
 
-  * Should be easy for others to copy + paste + run, **if they so choose**. Don't let inclusion of output break executability.
+  * Should be easy for others to copy + paste + run, **iff(if and only if) they so choose**. Don't let inclusion of output break executability.
```
Comment (batpigandme) [inline vignettes/reprex-dos-and-donts.Rmd]: Add a space between iff and parens?

Comment (jennybc) [reply vignettes/reprex-dos-and-donts.Rmd]: Let's just replace "iff" with "if and only if".

**Comment by jennybc** [comment]
Thanks!

**Pull request merged**